doc/source/whatsnew/v0.21.1.txt

```diff
@@ -57,6 +57,8 @@ Documentation Changes
 Bug Fixes
 ~~~~~~~~~
 - Bug in ``DataFrame.resample(...).apply(...)`` when there is a callable that returns different columns (:issue:`15169`)
+- Bug in ``pd.Series.rolling.skew()`` and ``rolling.kurt()`` with all equal values has floating issue (:issue:`18044`)
```

Comment:
producing numerical unstable results rather than NaN

Reply:
The original behavior for skewness  and kurtosis calculation is : 
if the {Xi} distribution is a uniform distribution ( all values are equal), Thus the variance of the values( Sum(Xi-Xmean) ^2 is zero. so B should be zero in this case. But some numerically unstable issue cause B not to be zero. 
so consider this situation, we assume if {Xi}'s variance is less than 1e-14, we treat it as uniform distribution.
based on previous behavior is return Nan. I don't want to break this behavior.
Some reference : 
from mathematica, for uniform distribution  skew/kurt return Nan
http://www.wolframalpha.com/input/?i=skewness%7B1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1%7D
http://www.wolframalpha.com/input/?i=kurtosis%7B1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1,1.1%7D